Documentation/Walkthrough.md

```diff
@@ -0,0 +1,206 @@
+# Getting started
+
+The definition and implementation of data and service contracts is the same as WCF. The major difference is in the definition of the host which is now based on ASP.NET Core, and the ceremony of how a service is exposed. The following is based on .NET 6, but the same steps apply to other versions of .NET.
+
+## Defining the service
+
+### 1. Create an ASP.NET Core Empty application, this provides the host for the service.
+
+*Visual Studio:*
+ ![Project Template](images/empty-web-project.png)
+
+*Command Line:*
+
+```cli
+mkdir CoreWCFDemoServer
+dotnet new web -n CoreWCFDemoServer -o CoreWCFDemoServer
+```
+
+### 1. Add references to the CoreWCF Nuget Packages
+
+*Visual Studio:*
+
+Using the package Manager console, add: 
+
+- CoreWCF.Primitives
+- CoreWCF.Http
+
+![Package Manager Console](images/nuget.png)
+
+*Command Line:*
+
+Edit the project file and add:
```

Comment:
For command line you can do:
```
dotnet add package CoreWCF.Primitives --version 1.0.0
dotnet add package CoreWCF.Http --version 1.0.0
```

I think you could miss out the version and it will grab the latest version which might be better.